.github/ISSUE_TEMPLATE/falsepositive.yml

```diff
@@ -0,0 +1,112 @@
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms
+name: False Positive Report
+description: File a Phishing report
+labels: ["False Positive"]
+assignees:
+  - mitchellkrogza
+  - funilrys
+body:
+  - type: markdown
+    attributes:
+      value: |
+        We understand being listed on a Phishing Database like this can be frustrating and embarrassing for many web site owners.
+
+        The first step is to remain calm.
+        
+        The second step is to rest assured one of our maintainers will address your issue as soon as possible.
+
+        Please make sure you have provided as much information details as possible to help speed up the process.
+
+  - type: textarea
+    id: record
+    attributes:
+      label: Domain/URL/IP(s) you believes NOT to be Phishing
+      description: |
+        Please list any domains and links listed here which you believe are a false positive.
+        
+        Please warn with "NSFW" where applicable.
+      placeholder: |
+        example.com
+        https://example.org/phishing
+        192.0.2.0/24
+      render: css
+    validations:
+      required: true
+
+  - type: checkboxes
+    id: source
+    attributes:
+      label: More Information
+      description: |
+        How did you discover your IP-address or domain was listed in the Phishing or Phishing.Database?
+      options:
+        - label: mitchellkrogza Phishing
+          required: false
+        - label: mitchellkrogza Phishing.Database
+          required: false
+        - label: Website was hacked
+          required: false
+        - label: Listed as Phishing on Phishtank or OpenPhish
+          required: false
+        - label: Listed on VirusTotal
+          required: false
+        - label: Other (Please tell us where in *Related external source* below)
+          required: false
+
+  - type: textarea
+    id: source_external
+    attributes:
+      label: Related external source
+      description: |
+        Please let us know where you found this if not listed above
```

Comment:
May be better:

```
If your source was not previously listed, please let us know here.
```

Reply:
Have added your line to https://github.com/spirillen/templatetest/commit/c5e6786f3f0a06da0e943d398035c617bdbd43c9 then we can test it there, before merging here-

Reply:
Latest chabge looks better on the active template